src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -429,14 +468,22 @@ public int compare(FeedbackQuestionAttributes q1, FeedbackQuestionAttributes q2)
         default:
             break;
         }
-        return recipients;
     }
     
     public boolean isQuestionHasResponses(String feedbackQuestionId) {
         return !frLogic.getFeedbackResponsesForQuestionWithinRange(feedbackQuestionId, 1)
                        .isEmpty();
     }
   
+    /**
+     * Checks if a question has been fully answered by a user.
+     * 
+     * @param question
+     * @param email
+     * @return {@code True} if there are no more recipients to give feedback to for the given
+     * {@code teamName}. {@code False} if not.
```

Comment:
You don't really need to `{@code}` true and false. Also consider "true if ..., false ~~if not~~ otherwise"